docs/index.rst

```diff
@@ -6,7 +6,35 @@
 Welcome to the documentation for the Axelrod Python library
 ===========================================================
 
-Here is quick overview of what can be done with the library.
+Here is quick overview of the current capabilities of the library:
+* Over 100 strategies from the literature and some exciting original
+contributions
+    * Classic strategies like TiT-For-Tat, WSLS, and variants
+    * Zero-Determinant and other Memory-One strategies
+    * Many generic strategies that can be used to define an array of popular
+    strategies, including finite state machines, strategies that hunt for
+    patterns in other strategies, and strategies that combine the effects of
+    many others
+    * Strategy transformers that augment that abilities of any strategy
+* Head-to-Head matches
+* Round Robin tournaments with a variety of options, including:
+    * noisy environments
+    * spatial games
+    * probabilistically chosen match lengths
+* Population dynamics
+    * The Moran process
+    * An ecological model
+*Multi-processor support, caching for deterministic interactions, automatically
+generate figures and statistics
+
+Every strategy is categorized on a number of dimensions, including:
+    * Deterministic or Stochastic
+    * How many rounds of history used
+    * Whether the strategy has access to the game matrix, the length of the
```

Comment:
whether the strategy makes use of?

The tournament can decide what it has access to right?